src/components/Dropdown/Dropdown.tsx

```diff
@@ -132,15 +134,25 @@ export const Dropdown: FC<DropdownProps> = ({
         : undefined;
 
     const showClear = !!activeItem && !!onClear;
-
-    const leftPosition =
-        triggerRef.current && overlayRef.current
-            ? triggerRef.current?.getBoundingClientRect().right - overlayRef.current?.getBoundingClientRect().width
-            : "auto";
-    const topPosition = triggerRef.current ? triggerRef.current?.getBoundingClientRect().top - 3 : "auto";
+    const dropdownRef = useRef<HTMLDivElement | null>(null);
+    const placement = `${position}-${alignment}`;
+
+    const popperInstance = usePopper(triggerRef?.current, dropdownRef.current, {
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore
+        placement: placement.toLowerCase(),
+        modifiers: [
```

Comment:
Currently, it popper flips the flyout automatically. Should we support that when defining it by property as well? Or would it be easier to add auto placements?

Reply:
@thisguyscode what do you think?

Reply:
Right! I see we could use placement auto as the dropdown is supposed to only go up or down. 
`placement: 'auto',
  modifiers: [
    {
      name: 'flip',
      options: {
        allowedAutoPlacements: ['top-left', 'top-right', 'bottom-left', 'bottom-right'], 
      },
    },
  ],`

Then there might be no need to have alignment and position props.